app/admin/views.py

```diff
@@ -23,6 +23,34 @@ def index():
     return render_template('admin/index.html')
 
 
+@admin.route('/view/all')
+@login_required
+@admin_required
+def listing_view_all():
+    """Search for listings"""
+    main_search_term = request.args.get('main-search', "", type=str)
+    favorite = True if request.args.get('favorite') == "on" else False
```

Comment:
Why does the admin have the option to filter something by `favorite`? It also doesn't look like it's an option on the form